### DISCUSSION TITLE
package manager like npm ？ yarn？cargo？

### DISCUSSION THREAD

**Discussion by ghost** [carbon-language/carbon-lang#1481] [General]
carbon language use what kind of package manager?
how to import other package?

i like npm/yarn/cargo

if there be similar solution, it will be very good.

I hate cpp do not have a official package manager.

**Comment by andrewfenn** [comment]
I think it's a pretty obvious and popular thing to have. I made some comments on this issue here: https://github.com/carbon-language/carbon-lang/issues/1573#issuecomment-1193260802

**Reply by LeeWlving** [reply]
I absolutely agree with your idea.

**Comment by YagaoDirac** [comment]
Yeah, Yarn and Anaconda are great.
